pkgs/misc/uboot/default.nix

```diff
@@ -126,6 +127,48 @@ let
       maintainers = with maintainers; [ bartsch dezgeg samueldr lopsided98 ];
     } // extraMeta;
   } // removeAttrs args [ "extraMeta" ]));
+
+  amlogic-boot-fip = fetchFromGitHub {
+    owner = "LibreELEC";
+    repo = "amlogic-boot-fip";
+    rev = "4369a138ca24c5ab932b8cbd1af4504570b709df";
+    sha256 = "sha256-mGRUwdh3nW4gBwWIYHJGjzkezHxABwcwk/1gVRis7Tc=";
+    meta.license = lib.licenses.unfreeRedistributableFirmware;
+  };
+
+  # Flashing instructions:
+  # dd if=${drv}/u-boot.bin.sd.bin of=$DEV conv=fsync,notrunc bs=512 skip=1 seek=1
+  # dd if=${drv}/u-boot.bin.sd.bin of=$DEV conv=fsync,notrunc bs=1 count=440
```

Comment:
If you have an expression for building SD image which uses this, you can put these into `sdImage.postBuildCommands`, check out https://github.com/novena-next/nixos-novena/blob/master/configuration.nix#L62-L64

The comment is fine of course, it's just something you might find interesting. :smile: